src/Native/String.js

```diff
@@ -203,75 +203,83 @@ function indexes(sub, str)
 	{
 		is.push(i);
 		i = i + subLen;
-	}	
-	
+	}
+
 	return _elm_lang$core$Native_List.fromArray(is);
 }
 
+
 function toInt(s)
 {
 	var len = s.length;
+
+	// if empty
 	if (len === 0)
 	{
-		return _elm_lang$core$Result$Err("could not convert string '" + s + "' to an Int" );
+		return intErr(s);
 	}
-	var start = 0;
-	if (s[0] === '-')
+
+	// if hex
+	var c = s[0];
+	if (c === '0')
 	{
-		if (len === 1)
+		// must be hex
+		if (s[1] !== 'x')
```

Comment:
If this is a number of minutes that I've extracted from a time-formatted string, like "09", it will fail. Is that desired behaviour?